src/components/dialogs/input-hooks.js

```diff
@@ -282,6 +284,7 @@ export const useConnectivityValue = ({
     }, [inputForm.toggleClear]);
 
     useEffect(() => {
+        dispatch(setFirstBuildNode());
```

Comment:
Why do we need this ?

Reply:
code not updated , ill do the cleaning and refactor